CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 ## Unreleased
 
 ### Fixed
+- Bug that occurrs when trying to add a user to an alert rule who had never been on any detection list before.
```

Comment:
Should always be a newline under markdown headers

Reply:
changed.